src/ec/packages/ec-component-timeline/timeline.html.twig

```diff
@@ -0,0 +1,96 @@
+{% spaceless %}
+
+{#
+Parameters:
+  - "toggleCollapsed": (string) (default: 'Show %d more items')
+  - "toggleExpanded": (string) (default: 'Show %d items')
+  - "items" (array) (default: []):
+    - "id": (string) (default: '')
+    - "label": (string) (default: '')
+    - "title": (string) (default: '')
+    - "content": (block) (default: '')
+  - "icon_path" (string) (default: ''): file containing the svg icons
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (array) (default: []): format:
+    - "name" (string) (default: '')
+    - "value" (string) (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-timeline2' %}
+{% set _extra_attributes = '' %}
+{% set _items = items|default([]) %}
+{% set _icon_path = icon_path|default('') %}
+{% set _toggle_collapsed = toggleCollapsed|default('Show %d more items') %}
+{% set _toggle_expanded = toggleExpanded|default('Hide %d items') %}
+
+
+{% set display_amount = items|length %}
+{% set hide_amount = 0 %}
+{% set bottom_amount = 0 %}
+
+{% if items|length >= 13 %}
+  {% set bottom_amount = 2 %}
+  {% set countbase = items|length - bottom_amount %}
+  {% set display_amount = 7 %}
+  {% if (countbase / 2) < 7 %}
+    {% set display_amount = countbase / 2 %}
+  {% endif %}
+  {% set hide_amount = countbase - display_amount %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<ol class="{{ _css_class }}"{{ _extra_attributes|raw }}
+    data-ecl-timeline="true">
+  {% if _items is not empty %}
+    {% for _item in _items %}
+      {# Display the show more button if required.#}
+      {% if hide_amount > 0 and loop.revindex == 2 %}
+        <li class="ecl-timeline2__item ecl-timeline2__item--toggle">
+          {% include '../ec-component-button/button.html.twig' with {
+            label: toggleCollapsed|replace({'%d': countbase - display_amount}),
+            variant: 'secondary',
+            type: 'button',
+            icon: {
+              path: _icon_path,
+              type: 'ui',
+              name: 'corner-arrow',
+              size: 'xs',
+            },
+            icon_extra_classes: 'ecl-icon--rotate-180',
```

Comment:
You could use the [`transform`](https://github.com/ec-europa/ecl-twig/blob/v2.7-dev/src/ec/packages/ec-component-icon/icon.html.twig#L11) parameter of the icon component instead of manually adding the class 😉